src/types/table.ts

```diff
@@ -76,3 +81,14 @@ export interface TablePaginationAttributes {
 }
 
 export type RowBulkAction = boolean | { enabled: boolean, disabledTooltip?: string }
+
+export interface SortHandlerFunctionParam {
+  /** The key of the column to be sorted */
+  key: string
+  /** The key of the column previously sorted */
+  prevKey: string
```

Comment:
question: is there a time when this is undefined? (e.g. on init?)

Reply:
It's never undefined, if there is no column that was previously sorted, this value is empty string (`''`)